services/ingestion/etl/cordis/csv/README.md

```diff
@@ -0,0 +1,27 @@
+# CORDIS ETL mapping
+
+Model to compare with is available at: https://ec-europa.github.io/eubfr-data-lake/
+
+| Field                | Target                |
+| -------------------- | --------------------- |
+| rcn                  |                       |
+| id                   | project_id            |
+| acronym              |                       |
+| status               | status                |
+| programme            | sub_programme_name    |
+| topics               |                       |
+| frameworkProgramme   | programme_name        |
+| title                | title                 |
+| startDate            | timeframe.from        |
+| endDate              | timeframe.to          |
+| projectUrl           | project_website       |
+| objective            | description           |
+| totalCost            | budget.total_cost     |
+| ecMaxContribution    | budget.eu_contrib     |
+| call                 |                       |
+| fundingScheme        | action                |
+| coordinator          | third_parties.name    |
+| coordinatorCountry   | third_parties.country |
+| participants         |                       |
+| participantCountries |                       |
```

Comment:
This one is `country_code` of `project_locations` or `country` in `third_parties`. The information I see in the file, however, is not `homogeneous`, it's mixed with `subjects`, we need to be careful.